packages/client/src/build-id-types.ts

```diff
@@ -0,0 +1,137 @@
+import { temporal } from '@temporalio/proto';
+
+/**
+ * Operations that can be passed to {@link WorkflowClient.updateWorkerBuildIdCompatability}.
+ *
+ * @experimental
+ */
+export type BuildIdOperation =
+  | NewIdInNewDefaultSet
+  | NewCompatibleVersion
+  | PromoteSetByBuildId
+  | PromoteBuildIdWithinSet
+  | MergeSets;
+
+/**
+ * This operation adds a new Build Id into a new set, which will be used as the default set for
+ * the queue. This means all new workflows will start on this Build Id.
+ *
+ * @experimental
+ */
+interface NewIdInNewDefaultSet {
+  operation: 'NEW_ID_IN_NEW_DEFAULT_SET';
```

Comment:
Looks like for schedules we made the action enum camel case. Let's do that here too.

Reply:
You led me astray in the airport!